Integrations/integration-PacketMail.yml

```diff
@@ -0,0 +1,118 @@
+description: Intel look up for IPS
+configuration:
+- display: API Key
+  name: apikey
+  defaultvalue: ""
+  type: 4
+  required: false
+- display: Use Proxy
+  name: proxy
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: CSV list of lists that should be considered malicious
+  name: maliciousList
+  defaultvalue: cleanmx_virus,et_c2
+  type: 0
+  required: false
+script:
+  script: |+
+    var initialurl = 'https://www.packetmail.net/iprep.php/';
+    var malLists = params.maliciousList.split(',');
+    for (i = 0 ; i < malLists.length; i++) {
+        malLists[i] = malLists[i].trim();
+    }
+
+    var sendRequest = function(url, param) {
+
+        // handle '/' at the end of the url
+        if (url[url.length - 1] === '/') {
+            url = url.substring(0, url.length - 1);
+        }
+
+        var res = http(
+            url,
+            {
+                Method: 'GET', // Can be POST, PUT, DELETE, HEAD, OPTIONS or CONNECT
+                Headers: {
+                    Accept: ['application/json']
+                }
+            },
+            true,
+            params.proxy
+        );
+
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+
+        return JSON.parse(res.Body);
```

Comment:
add response content type check for json. some cases server return 200 with error in the html body.